public/grpc-client.js

```diff
@@ -114,6 +133,14 @@ module.exports.init = async function({
     lnd[method](body, { deadline }, handleResponse);
   });
 
+  ipcMain.on('lndAtplRequest', (event, { method, body = {} }) => {
+    const deadline = new Date(new Date().getTime() + GRPC_TIMEOUT);
```

Comment:
I don't actually think we use these `deadline` dates anywhere. We can remove them from all the calls ... also above for unlocker/lnd events.

Reply:
fixed